lms/static/sass/multicourse/_entitlement_dashboard.scss

```diff
@@ -0,0 +1,67 @@
+.entitlement-actions-wrapper {
+    @extend .wrapper-action-more;
+
+    .entitlement-action {
+        @extend .action;
+    }
+
+    .entitlement-action-more {
+        @extend .action-more;
+    }
+
+    .entitlement-actions-dropdown {
+        @extend .actions-dropdown;
+
+        .entitlement-actions-dropdown-list {
+            @extend .actions-dropdown-list;
+
+            .entitlement-actions-item {
+                @extend .actions-item;
+            }
+        }
+    }
+}
+
+.entitlement-unenrollment-modal {
+    @extend .modal;
+    margin-top: -60px;
+
+    .entitlement-unenrollment-modal-inner-wrapper {
+        @extend .inner-wrapper;
+
+        .entitlement-unenrollment-modal-close-btn {
+            @extend .close-modal;
+        }
+
+        .entitlement-unenrollment-modal-header {
+            @extend .unenroll-header;
+        }
+
+        .entitlement-unenrollment-modal-error-text {
+            @extend .modal-form-error;
+            background: tint($red, 95%);
+            margin-left: $baseline;
+            margin-right: $baseline;
```

Comment:
margin: 0 $baseline;

Reply:
I want the inherited margin-top/margin-bottom values to apply, so I think I will need to leave this as is.